app/api/images/images.controller.js

```diff
@@ -490,6 +493,23 @@ exports.getGeoreferencers = utils.route(async (req, res) => {
   res.status(200).send(topUsers);
 });
 
+exports.checkWaitingValidation = utils.route(async (req, res) => {
+  const image_id = req.params.id;
+
+  const whereClause = {
+    image_id: image_id,
+    state: 'waiting_validation'
+  }
+
+  const hasWaitingValidationGeolocalisation = await models.geolocalisations.findOne({
```

Comment:
findOne!! so it will return one entry if exist

Reply:
Explanation in the others comment